improver_tests/synthetic_data/test_set_up_test_cubes.py

```diff
@@ -627,6 +628,13 @@ def test_standard_grid_metadata(self):
         self.assertEqual(result.attributes["mosg__grid_domain"], "uk_extended")
         self.assertEqual(result.attributes["mosg__model_configuration"], "uk_ens")
 
+    def test_single_percentile(self):
+        """Test a cube with one percentile correctly stores this as a scalar
+        coordinate"""
+        result = set_up_percentile_cube(self.data[1:2], self.percentiles[1:2])
+        dim_coords = get_dim_coord_names(result)
```

Comment:
Why do you use `get_dim_coord_names` here, but not in `test_Percentile_converter.py`?

Reply:
No particular reason - mainly for consistency with the surrounding code in `test_PercentileConverter`.  But I can change this?

Reply:
I'd rather change the other one, see comment in `test_PercentileConverter`